copy/copy.go

```diff
@@ -83,10 +86,23 @@ func (d *digestingReader) Read(p []byte) (int, error) {
 type Options struct {
 	RemoveSignatures bool   // Remove any pre-existing signatures. SignBy will still add a new signature.
 	SignBy           string // If non-empty, asks for a signature to be added during the copy, and specifies a key ID, as accepted by signature.NewGPGSigningMechanism().SignDockerManifest(),
+	ReportWriter     io.Writer
```

Comment:
We make more assumptions than just an `io.Writer`, especially due to `pb.v1` expecting a TTY which supports `'\r'` for overwriting. `InteractiveProgressTTY` or something?